.github/workflows/build-and-test.sh

```diff
@@ -8,7 +8,7 @@ export DEBIAN_FRONTEND=noninteractive
 export ROS_PYTHON_VERSION=3
 
 apt update -qq
-apt install -qq -y lsb-release wget curl build-essential
+apt install -qq -y lsb-release wget curl build-essential git
```

Comment:
is `git` needed?

Reply:
Ah, no, it was in a previous attempt. Removed